### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by BusyJay**
Fix stale instruction and add crates badges.

This pr should only be merged after 0.1.0 version is released.

### CODE REVIEW

**Comment by siddontang** [comment]
CI failed

**Comment by siddontang** [comment]
PTAL @queenypingcap

**Comment by siddontang** [comment]
LGTM

PTAL @overvenus

File: README.md [hunk 1]
```diff
@@ -64,8 +72,8 @@ $ protoc --rust_out=. --grpc_out=. --plugin=protoc-gen-grpc=`which grpc_rust_plu
 To include this project as a dependency:
 
 ```
-[dependencies.grpc]
-git = "https://github.com/pingcap/grpc-rs.git"
+[dependencies]
+grpcio = "0.1.0"
```
Comment (overvenus) [inline README.md]: I prefer not to include the PATCH version `0`.

**Comment by siddontang** [comment]
PTAL @overvenus

**Comment by BusyJay** [comment]
The documentation can't be generated on docs.rs currently, let's remove it for now. PTAL

**Comment by siddontang** [comment]
LGTM

**Pull request merged**